packages/lesswrong/styles/_collections.scss

```diff
@@ -28,12 +28,25 @@ $card-padding: 20px;
 .collection-card-item {
   position:relative;
   display: inline-block;
+
+  @include max-small() {
+    margin:10px calc(50% - 180px);
```

Comment:
Hmm, I would really prefer to avoid using calc because of mobile browser compatibility. Not like our compatibility overall is great, but I think it doesn't work at all on the default android browser and mobile safari.